README.md

```diff
@@ -83,7 +83,33 @@ The lint task will check to see that SVG images are optimized. To optimize image
 npm run optimize-assets
 ```
 
-To build **Help Center** pages, follow the guidelines in the [Help README](_help/README.md).
+### NetlifyCMS
+[NetlifyCMS](https://www.netlifycms.org/) is an open source content management system that we use to edit content on the brochure site. To develop and make changes to the CMS, or to edit content locally, first comment out the following lines in `admin/config.yml`
+
+```
+backend:
+  repo: 18f/identity-site
+  branch: main
+  base_url: https://federalistapp.18f.gov
+  auth_endpoint: external/auth/github
+  preview_context: federalist/build
+  use_graphql: true
+```
+and uncomment the following lines, making sure that you change the name of your branch
+```
+backend:
+  name: git-gateway
+  branch: YOUR-BRANCH-NAME-HERE
+  proxy_url: http://localhost:8081/api/v1
+local_backend: true
```

Comment:
since the contents are in the YML file already, what if we just reference the section? That way if the yml file changes we don't need to make sure to keep this section up to date to match it?